src/common/classes/ClumpletReader.cpp

```diff
@@ -62,11 +62,11 @@ class ClumpletDump : public ClumpletReader
 		return t2;
 	}
 protected:
-	virtual void usage_mistake(const char* what) const
+	[[noreturn]] virtual void usage_mistake(const char* what) const
```

Comment:
I've thought we decided not to mark virtual functions as [[noreturn]].

Reply:
I must have missed that one when I removed it.